### PR TITLE
Broken link Airbnb

### PR DESCRIPTION
**Pull request by Unstop-M**
Fixing broken link

### CODE REVIEW

File: _data/apps.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 - name: Airbnb
   featured: true
-  url: https://www.airbnb.co://www.airbnb.com
+  url: https://www.airbnb.com
```
Comment (mctaylorpants) [inline _data/apps.yml:39]: Thanks for catching that!

File: _data/apps.yml [hunk 2]
```diff
@@ -128,3 +128,13 @@
...
+
+- name: Lobsters
+  url: https://lobste.rs
+  logo_url: https://pbs.twimg.com/profile_images/917080328400814080/NVFfU2GQ_400x400.jpg
+  business_description: >
+    Lobsters is a technology-focused community centered around link aggregation and discussion. Launched on July 3rd, 2012, it was created by joshua stein with some specific design goals in mind to avoid problems faced by other link aggregation sites.
+  rails_info:
+    version: 5.1
+    description: >
+      Rails code running the lobste.rs link aggregation site
```
Comment (mctaylorpants) [inline _data/apps.yml]: Is it OK if we reword this a little bit? Something like, "Rails runs the lobste.rs link aggregation site."

**Review by mctaylorpants — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: _data/apps.yml (hunk 2 above)
Comment (Unstop-M) [reply _data/apps.yml]: Okey

**Pull request merged**